src/PkgDeps.jl

```diff
@@ -128,4 +115,43 @@ function users(pkg_name::String, pkg_registry::RegistryInstance; kwargs...)
     return users(uuid; kwargs...)
 end
 
+"""
+    direct_dependencies(pkg_name::String;
+        registries::Array{RegistryInstance}=reachable_registries()) -> Dict{String, UUID}
+    direct_dependencies(pkg_uuid::UUID; registries::Array{RegistryInstance}=reachable_registries()) -> Dict{String, UUID}
+    
+Returns the direct dependencies of the latest version of a given package
+in the form of `Dict` with names as keys and UUIDs as values.
+"""
+direct_dependencies
+
+function direct_dependencies(pkg_entry::PkgEntry)
+    base_path = joinpath(pkg_entry.registry_path, pkg_entry.path)
+    deps_file_path = joinpath(base_path, "Deps.toml")
+    all_direct_dependencies = Dict{String, UUID}()
+    if isfile(deps_file_path)
+        deps_content = parsefile(deps_file_path)
+        dependency_versions = collect(keys(deps_content))
+        latest_version = _get_latest_version(base_path)
+        # Use the latest_version of pkg, and check to see if pkg_name is in its dependents
+        for version_range in dependency_versions
+            if in(latest_version, VersionRange(version_range))
+                for (k, v) in pairs(deps_content[version_range])
+                    all_direct_dependencies[k] = UUID(v)
+                end
+            end
+        end
+    end
+    return all_direct_dependencies
+end
+
+function direct_dependencies(pkg_name::String; registries::Array{RegistryInstance}=reachable_registries())
+    return direct_dependencies(_find_latest_pkg_entry(pkg_name, missing; registries))
```

Comment:
Could just be one-liner functions since we're just wrapping around the other one.